modules/contact/src/PenetrationInfoAuxAction.C

```diff
@@ -0,0 +1,90 @@
+/****************************************************************/
```

Comment:
Why not just do this in the ContactAction along with everything else?  I guess it's just a personal taste thing, but I think it's confusing to have a bunch of separate Action classes to set up different things.